newrelic-notify.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -ex pipefail
+
+NEWRELIC_USER=gitlab
+# TODO: where/how to get newrelic api key?
+# retrieve from somewhere on disk or a service?
+API_KEY=$(< ~/.newrelic-api-key)
+
+# get the image value from the base deployment matching the K8S_NAMESPACE
+FULL_IMAGE="kubectl get deploy ${K8S_NAMESPACE} -n ${K8S_NAMESPACE} -o jsonpath=\"{$.spec.template.spec.containers[0].image}\""
+
+# last 7 characters of the image value will be the hash
+GIT_COMMIT_SHORT="${FULL_IMAGE: -7}"
```

Comment:
You can also get `GIT_COMMIT_SHORT` from `web-deploy.yaml` using regex

`grep -oP '(image: .+?:)\K.*' web-deploy.yaml`

and in this case you'd call `./newrelic-notify.sh $(grep -oP '(image: .+?:)\K.*' web-deploy.yaml)`

This avoids talking to k8s and adding another env variable in the gitlab config.

regex ftw! :fist_raised: